lib/ansible/modules/cloud/vmware/vmware_guest_disk.py

```diff
@@ -108,6 +108,20 @@
      - '   If C(state) is set to C(absent), disk will be removed permanently from virtual machine configuration and from VMware storage.'
      - '   If C(state) is set to C(present), disk will be added if not present at given SCSI Controller and Unit Number.'
      - '   If C(state) is set to C(present) and disk exists with different size, disk size is increased.'
+     - ' - C(iolimit) is set, disk will be added with the values specified under iolimit section: Valid values are:'
+     - '     - C(limit)(long) limit value to be given, by default it goes to custom.'
```

Comment:
Could you please explore if `suboptions` can be used to here ? I am not sure about this. You can find example [here](https://github.com/ansible/ansible/blob/548fa65ac6db1f3b24944a106fa01872362fea17/lib/ansible/modules/cloud/amazon/aws_batch_job_definition.py#L85)